modules/reference/pages/properties.adoc

```diff
@@ -0,0 +1,6361 @@
+= Redpanda Configuration Properties 
+:description: Redpanda configuration properties. 
+
+== Broker
+
+Broker configuration properties are applied individually to each broker in a cluster. 
+
+IMPORTANT: After you change a broker-level property setting, you must restart the broker for the change to take effect. 
+
+---
+
+=== admin
+
+Address and port of admin server
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/model::broker_endpoint
+
+*Default:* '{model::broker_endpoint(net::unresolved_address("127.0.0.1", 9644))}'
+
+=== admin_api_doc_dir
+
+Admin API doc directory
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* '/usr/share/redpanda/admin-api-doc'
+
+=== admin_api_tls
+
+TLS configuration for admin HTTP server
```

Comment:
Specifies the TLS configuration for the Admin API.

Reply:
**ENG:** Please clarify whether this should use Admin API (in the title of the property) or admin HTTP server (what is currently in the code).